### PR TITLE
Improve binders doc

### PR DESCRIPTION
**Pull request by jccazeaux**
See #637

### CODE REVIEW

File: docs/docs/guide/_sections/binders/two-way.md [hunk 1]
```diff
@@ -3,27 +3,28 @@ Two-way binders, like one-way binders, can update the DOM when a model property
...
-  bind: function(el) {
-    adapter = this.config.adapters[this.key.interface]
-    model = this.model
-    keypath = this.keypath
-
-    this.callback = function() {
-      value = adapter.read(model, keypath)
-      adapter.publish(model, keypath, !value)
-    }
+rivets.binders['mobi-calendar'] = {
```
Comment (Leeds-eBooks) [inline docs/docs/guide/_sections/binders/two-way.md]: Isn't this quite a specific example? Should we keep the old `toggle` example too, as it is more generic?

**Comment by jccazeaux** [comment]
Yes it may stay as a "one-way" example. I'll write this tomorrow

File: docs/docs/guide/_sections/binders/two-way.md (hunk 1 above)
Comment (jccazeaux) [inline docs/docs/guide/_sections/binders/two-way.md]: Toggle example was buggy as explained in #634 
I found the calendar example more explicit for a two way binder as it used all API including `getValue`. 
Toggle example is be more generic but it has to be reworked.

**Comment by Leeds-eBooks** [comment]
I'm going to merge this as the current example is simply wrong. We can revisit this if people find the new example confusing.

**Comment by Leeds-eBooks** [comment]
Oh, should this be merged into `gh-pages`?

**Comment by jccazeaux** [comment]
It's already on the PR #653 
Doc in `master` and `gh-pages` will be sync when this PR is merged.

**Pull request merged**